packages/components/FocusZone/RCTFocusZone.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name         = 'FluentUI-React-Native-FocusZone'
+  s.name         = 'RCTFocusZone'
```

Comment:
@chripear @chiuam I changed the podspecs' names to match the component they represent. This means FocusZone is an odd one out, the only one with the prefix "RCT". @HeyImChris has told me that RCT is reserved for React Native internal types, but we (FURN / rn-win32 ) have used it for FocusZone. How would you all feel about either:

1) Keeping the naming as is in this change, where the pod spec matches the component, and the internal objective C object is RCTFocusZone
2) Changing the podspec to "FRNFocusZone", and keeping the internal objective c object RCTFocusZone to keep binary compatibility
3) Changing both the podspec and the objective C type to "FRNFocusZone", introducing a divergence (win32 has a native module named "RCTFocusZone" while macOS has a native module "FRNFocusZone" and updating it downstream so that we use the consume the correct FRNFocusZone type?

Reply:
I'm gonna merge this and leave this as a followup question for expediency's sake. Sticking with 1)